service/src/main/java/uk/nhs/adaptors/gp2gp/common/mongo/ttl/MongoTtlCreator.java

```diff
@@ -0,0 +1,41 @@
+package uk.nhs.adaptors.gp2gp.common.mongo.ttl;
+
+import java.time.Duration;
+import java.util.Optional;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.data.domain.Sort;
+import org.springframework.data.mongodb.core.index.Index;
+import org.springframework.data.mongodb.core.index.IndexInfo;
+import org.springframework.data.mongodb.core.index.IndexOperations;
+
+@Slf4j
+public class MongoTtlCreator extends TtlCreator {
+    public static final String TTL_INDEX_NAME = "TTL";
+    private static final String FIELD_KEY = "translationTimestamp";
```

Comment:
Our EhrExtractStatus has no translationTimestamp.

This index should be on the field "updatedAt"